index.php

```diff
@@ -0,0 +1,57 @@
+<?php
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link href="https://fonts.googleapis.com/css?family=Cantarell:400,700" rel="stylesheet">
+    <link rel="stylesheet" href="css/normalize.css">
+    <link rel="stylesheet" href="css/index.css">
+    <title>Pokédex</title>
+</head>
+<body>
+    <main>
+        <form action="index.php" class="nameSearch">
```

Comment:
This should probably be lined up with the ul beneath it and every element in the page should be the same size, as well as styled. Compare with the form elements on the login page.

Reply:
Changed in e094b46.